src/vcpkg/binarycaching.cpp

```diff
@@ -253,7 +253,6 @@ namespace
     {
         nothing,
         always,
-        on_fail,
```

Comment:
Is this removed due to no mechanism to access it via UX?

Reply:
That's correct. There was one path where it got set but that path seemed like a bug. See rationale in https://github.com/microsoft/vcpkg-tool/pull/1339/commits/8b30b0cf3100bf58cc1f0ba6b28371e1f174ce89